src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial.mdx

```diff
@@ -0,0 +1,476 @@
+
+import queriesnrqltutorial9 from 'images/queries-nrql_screenshot-crop-tutorial-9.png'
+
+import queriesnrqltutorial10 from 'images/queries-nrql_screenshot-crop-tutorial-10.png'
+
+import queriesnrqltutorial11 from 'images/queries-nrql_screenshot-crop-tutorial-11.png'
+
+import queriesnrqltutorial12 from 'images/queries-nrql_screenshot-crop-tutorial-12.png'
+
+import queriesnrqltutorial13 from 'images/queries-nrql_screenshot-crop-tutorial-13.png'
+
+import queriesnrqltutorial14 from 'images/queries-nrql_screenshot-crop-tutorial-14.png'
+
+import queriesnrqltutorial15 from 'images/queries-nrql_screenshot-crop-tutorial-15.png'
+
+import queriesnrqltutorial16 from 'images/queries-nrql_screenshot-crop-tutorial-16.png'
+
+import queriesnrqltutorial17 from 'images/queries-nrql_screenshot-crop-tutorial-17.png'
+
+import queriesnrqltutorial18 from 'images/queries-nrql_screenshot-crop-tutorial-18.png'
+
+import queriesnrqltutorial19 from 'images/queries-nrql_screenshot-crop-tutorial-19.png'
+
+import queriesnrqltutorial20 from 'images/queries-nrql_screenshot-crop-tutorial-20.png'
+
+We've covered the basics of NRQL in our other topics, such as [what NRQL is](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/), [how NRQL works](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works/), and [how NRQL relates to charts and dashboards](docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql/). Now that you're familiar with some of the basic NRQL syntax, you're ready to try the examples in this tutorial! Below we'll cover some foundational concepts within the language and help you create some basic NRQL queries using your data. This tutorial teaches you how to:
+
+* Use `SELECT` and `FROM`
+* Use `LIMIT`
+* Specify attributes
+* Aggregate using `average()`, `max()`, `min()`, `sum()`, and `count()`
+* Use `SINCE` and `UNTIL`
+* Plot line graphs with `TIMESERIES`
+* Filter using `WHERE`
+* Group attributes using `FACET`
+
+Want to follow this tutorial within the New Relic platform? We've included a NRQL Lessons application on our [opensource.newrelic.com](https://opensource.newrelic.com/projects/newrelic/nr1-learn-nrql) with the same steps and examples to help you quickly see NRQL's value and power within the application. The video below shows you how to install the NRQL Lesson.
+
+<Video
+  id="3UHEHbprs6I"
+  type="youtube"
+/>
+
+Ready to get started? Let's go!
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="first-query"
+    title="1. Your first query"
+  >
+  <SideBySide>
+  <Side>
+Let's start by looking at a single event type in NRDB called **Transaction**, gathered by New Relic APM. Every NRQL query must have `SELECT` and `FROM` clauses: you must `SELECT` some data and tell us where it's `FROM`. Start with this basic query that selects everything from the **Transaction** event type.
```

Comment:
I'd recommend using code format instead of bold, so `Transaction` instead of **Transaction**. 

Instead of 'selects everything', as that may be ambiguous as to what that means, maybe 'returns the`Transaction` events reported in your New Relic account' or something.

Reply:
Roger, I'll get this change made. Thank you!